### PR TITLE
fix(web): generate nonce on every request (BOAS-1597)

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
## Describe your changes

- Amended both appeals and apps to create a nonce on every request
- Manually tested the functionality

## BOAS-1597 hard coded nonce
https://pins-ds.atlassian.net/browse/BOAS-1597

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/app/app.express.js [hunk 1]
```diff
@@ -53,16 +54,34 @@ app.use(bodyParser.json());
...
-			scriptSrc: ["'self'", () => `'nonce-${locals.cspNonce}'`],
-			defaultSrc: ["'self'", config.blobStorageUrl],
-			'font-src': ["'self'"],
-			'img-src': ["'self'", config.blobStorageUrl],
-			'style-src': ["'self'"]
+	helmet({
+		contentSecurityPolicy: {
+			directives: {
+				// @ts-ignore
+				scriptSrc: ["'self'", "'unsafe-eval'", addCSPNonce],
```
Comment (beejjacobs) [inline appeals/web/src/server/app/app.express.js:79]: We don't need 'unsafe-eval'

File: apps/web/src/server/app/app.express.js [hunk 2]
```diff
@@ -53,27 +54,45 @@ app.use(bodyParser.json());
...
-				"'sha256-aqNNdDLnnrDOnTNdkJpYlAxKVJtLt9CtFLklmInuUAE='",
-				// these are for the html-content-editor warning message show/hide logic
-				"'sha256-dCNOmK/nSY+12vHzasLXiswzlGT5UHA7jAGYkvmCuQs='",
-				"'sha256-ZdHxw9eWtnxUb3mk6tBS+gIiVUPE3pGM470keHPDFlE='"
-			]
+	helmet({
+		contentSecurityPolicy: {
+			directives: {
+				// @ts-ignore
+				scriptSrc: ["'self'", "'unsafe-eval'", addCSPNonce],
```
Comment (beejjacobs) [inline apps/web/src/server/app/app.express.js:79]: We don't need 'unsafe-eval'

**Review by beejjacobs — CHANGES_REQUESTED** [review]
Looks good! One little change we don't want, but otherwise good.

File: appeals/web/src/server/app/app.express.js (hunk 1 above)
Comment (BenSurgisonPINS) [reply appeals/web/src/server/app/app.express.js]: Removed as requested

File: apps/web/src/server/app/app.express.js (hunk 2 above)
Comment (BenSurgisonPINS) [reply apps/web/src/server/app/app.express.js]: Removed as requested, retested and still works

**Review by beejjacobs — APPROVED** [review]
LGTM!

**Pull request merged**